### ISSUE TITLE
[Discussion] Gitea/Forgejo

### ISSUE THREAD

**Issue by wkeiuluf** [thomiceli/opengist#154]
I don't really know the best way to handle this...but thought I would bring it up for discussion.

---

## Context
[Forgejo](https://codeberg.org/forgejo/forgejo) is a soft-fork of [Gitea](https://github.com/go-gitea/gitea). They provide [a justification](https://forgejo.org/faq/#why-was-forgejo-created) for existing:

> **Why was Forgejo created?**
> 
> In October 2022 the domains and trademark of Gitea were transferred to a for-profit company without knowledge or approval of the community. Despite [writing an open letter](https://gitea-open-letter.coding.social/), the takeover was later confirmed. Forgejo was created as an alternative providing a software forge whose governance further the interest of the general public.
> 
> Forgejo was initially presented as a “soft-fork” of Gitea, similar to [LineageOS](https://lineageos.org/), a community led distribution based on Android from Google. It is however better described as a product built on top of Gitea, Git and hundreds of other Free Software projects. Forgejo also has its own documentation, infrastructure, release pipeline, CI infrastructure, distribution channels etc.

## Why did you open this discussion issue, @wkeiuluf?
As you may or may not know, users aren't always the brightest. Seeing "login with gitea" might be confusing to a user (not the admin setting everything up, but one of their users) that is like "hey, what's gitea?"

Basically, I think it would be nice to have an aliased set of `GITEA` stuff but with `FORGEJO` in place. But that might not be the cleanest way to handle it. Hell, people reading this may even just think "you're dumb, who cares" and that's fine, too; that's why I opened up a discussion for this. 😅

Another idea: maybe just change the button to "login with git" or something?

---

Anyway, just curious other people's thoughts on it.

**Comment by thomiceli** [comment]
I did not mess with Forgejo yet, but I guess since it's a fork, the oauth system works as same as Gitea, so it's more like a semantic problem ?

**Comment by crapStone** [comment]
I opened a PR that let the server admin decide what name the button should have. I think it's ok for server admins to know that Gitea and Forgejo can be used interchangeably as long as the APIs do not break compatibility. And when that happens we have to implement another integration anyway :man_shrugging:.
